src/main/java/kr/pickple/back/crew/controller/CrewController.java

```diff
@@ -76,4 +77,15 @@ public ResponseEntity<Void> permitCrewMemberShip(
         return ResponseEntity.status(NO_CONTENT)
                 .build();
     }
+
+    @DeleteMapping("/{crewId}/members/{memberId}")
+    public ResponseEntity<Void> deleteCrewMemberShip(
```

Comment:
`p2`

현재 해당 API는 상태가 `신청`인 상태의 CrewMember를 지우는 것인데 deleteCrewMemberShip이라고 되어있어서 `신청`목록중에서 삭제를 하는 것인지 아니면 `확정`인 상태에서 해당 크루원을 삭제하는 것인지 구분이 되지 않는 것 같습니다!

메소드 명을 아래와 바꾸는건 어떻게 생각하시나요?! 바꾸신다면 서비스 레이어에 있는 메서드도 동일하게 바꿔주시면 좋을 것 같습니다!
```suggestion
    public ResponseEntity<Void> deleteWaitingCrewMember(
```

Reply:
저는 생각이 반대인데, 사실 이 부분은 저도 게스트 모집 도메인에서 그냥 deleteGameMember 라고 했었는데요.
지금이야 물론 "신청" 목록에서 지우는 기능만 MVP로 되어 있어서 그렇지만
결국은 크루장이 해당 크루원을 탈퇴 시키는 기능도 들어갈텐데, 그러면 "확정" 상태의 크루원도 지워져야 할 것입니다.
따라서 반드시 이 메서드가 "신청" 상태만을 삭제해야 하는 것 보다는, 상태와는 상관없이 삭제처리 하는게 더 맞다고 생각해요!
결국은 `"신청" 상태에 대한 거절, 취소` 와 `"확정" 상태에 대한 강퇴, 탈퇴`가 crewMember 테이블에서 레코드를 hard delete 하는 똑같은 기능을 요구하기 때문이죠.

Reply:
그럼 "신청" 상태에 대한 거절, 취소와 "확정" 상태에 대한 강퇴, 탈퇴의 경우 동일한 기능이기에 동일한 API로 가져가고 추후 확정에 대한 강퇴, 탈퇴 기능이 추가 될 때 파라미터나 body 값으로 신청 상태에 대한 것인지 확정 상태에 대한 것인지 전달해주어야겠네요!

확장성을 고려하고 위와같이 구현한다면 바꾸지 않아도 될 것 같습니다~!

Reply:
네~ 저도 해당 사항을 조금 고민했었는데요~ 추후의 확장성을 위해 그대로 가겠습니다~